test-clients/src/main/java/com/hedera/services/legacy/regression/ProtectedAccountUpdateTests.java

```diff
@@ -245,13 +258,16 @@ public void updateAccountWithMasterAsPayer() throws Throwable {
           log.info("update failed: payer=" + payerID.getAccountNum() + ", target acount=" + account);
         }
         else if(account.getAccountNum() >= 51 && account.getAccountNum() <= 80)  // master can update protected accounts 51 through 80 without needing sig of updated account
-          updateAccount(account, payerID, nodeID, ResponseCodeEnum.OK, ResponseCodeEnum.SUCCESS, false, null);
-        else if(account.getAccountNum() <= 1000) { // master can NOT update protected accounts other than 51 through 80
+		{
```

Comment:
This brace should be next to `else if {`

Reply:
Updated, Thanks